ultranest/popstepsampler.py

```diff
@@ -536,8 +536,219 @@ def __next__(
         else:
             return None, None, None, nc
 
+class PopulationSimpleSliceSampler():
+    """
+       Vectorized Slice sampler without stepping out procedure.
+       In Comparison the PopulationSliceSampler, the sampler calls the likelihood on
+       batch of points of the same size.
+
+       Sliced are defined by the generate_direction function on a interval defined
+       around the current point. The centred interval has the width of the scale parameter.
```

Comment:
"The centred interval has the width of the scale parameter, i.e, there is no stepping out procedure as in :py:class:PopulationSliceSampler."

Reply:
Please add here a clear warning (maybe a separate paragraph), that because the slice length is guessed rather than ensured to cover the full length by a stepping out procedure, detailed balance is probably not preserved, so this sampler should be used with caution.